HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -992,7 +994,7 @@ method:: isValidUGenInput
 returns:: false if receiver cannot be used in UGen.
 
 
-SECTION::Special Functions
+subsection::Special Functions
```

Comment:
This solves the problem where the methods show up after the undocumented methods (#3694), but it would be nice to retain the bold emphasis of a section head for this. I don't feel very strongly about this though...

Relatedly, if functional notation is used (see comment below), `subsection::` can't be used here, otherwise the file won't render:
```
METHOD argument string is not allowed inside CLASSMETHODS or INSTANCEMETHODS
```
so this change could only remain if sticking with receiver notation.

Reply:
OK, I see. So there is something broken in the help system, and unfortunately I have no time to look into a fix.

This produces a trade off: should the methods be navigatable by the table of contents *or* should they be documented in the more adequate functional notation?

I am unsure - if you like, I can revert that one commit.

Reply:
If I were to choose I would prefer the functional notation, in anticipation of the help system eventually being fixed. Luckily the `subsection::` anchor links still work in the table of contents.

Reply:
ok. so that would mean I revert 979dd763f0ce9717fdd95b61b3a78d79f5739d73 and then add the line back in:
`subsection::Special Functions` ?

Reply:
Not quite: revert 979dd763f0ce9717fdd95b61b3a78d79f5739d73, which will restore `section::Special Functions`. This is required in order to allow the following `method::` tags. 

If it were `subsection::Special Functions`, these methods would be interpreted as being in the section `instancemethods::`, in which the `method::` tag apparently isn't allowed and throws an error.

Reverting this in turn requires you to revert 4d25dc2a3974d079ee2918449a3ec932550acf41, so the methods are copied.

Reply:
But before 4d25dc2 the method copying didn't work at all (at least for me). Are you sure it worked?

Reply:
It worked for me, I'm not sure why it wouldn't work for you.
If using functional notation under the `subsection::` tag means that the "section" is still `instancemethods::`, and I get the error that I posted in my first comment in this thread. So using the `section::` tag was the solution to this.

If the method copying is the hangup, we could simply do without it for now, it was really just a convenience.  If we got it building the help files, it would still not be working properly anyway. I'd be happy to revisit this after (#3694) is addressed.

If you want to resolve this discrepancy before merging,  I unfortunately won't have time to have another look at this for a few days.

Reply:
Hi @telephon 

After working with these methods for a while, I'm seeing that despite my preference for functional notation, it's quite painful that the linking is broken, so I'm of the mind now to say let's go ahead with the changes as you've proposed them in this PR. 

After this merge I may submit a PR that adds the functional notation in the description of each method where appropriate.

So I'm for merging this now, especially considering you've made some other important corrections with this PR. What do you think?

Reply:
good!